css/main.css

```diff
@@ -22,10 +21,6 @@
     justify-content: center;
 }
 
-.hero .illustration {
```

Comment:
Removing this makes the illustration disapear on my browser (Firefox V62)
<img width="1680" alt="capture d ecran 2018-09-24 a 14 02 31" src="https://user-images.githubusercontent.com/13916213/45951155-bbeae080-c002-11e8-8aeb-e5997f0ca071.png">